checklists/hci_checklist.en.json

```diff
@@ -0,0 +1,504 @@
+{
+  "items": [
+    {
+      "category": "Storage",
+      "subcategory": "Physical",
+      "text": "Is all planned storage pool storage direct attached (SATA, SAS, NVMe)?",
+      "waf": "Performance",
+      "guid": "9f519499-5820-4060-88fe-cab4538c9dd0",
+      "id": "01.01.01",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/windows-server/storage/storage-spaces/storage-spaces-direct-hardware-requirements"
+    },
+    {
+      "category": "Storage",
+      "subcategory": "Physical",
+      "text": "Are disks symmetrical across all nodes?",
+      "waf": "Performance",
+      "guid": "f7c015e0-7d97-4283-b006-567afeb2b5ca",
+      "id": "01.01.02",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure-stack/hci/concepts/drive-symmetry-considerations#understand-capacity-imbalance"
+    },
+    {
+      "category": "Storage",
+      "subcategory": "S2D",
+      "text": "Is parity disk redundancy only used for low I/O volumes (backup/archive)?",
+      "waf": "Performance",
+      "guid": "f785b143-2c1e-4466-9baa-dde8ba4c7aaa",
+      "id": "01.02.01",
+      "severity": "Medium",
+      "link": "Fault tolerance and storage efficiency on Azure Stack HCI and Windows Server clusters - Azure Stack HCI | Microsoft Learn"
```

Comment:
this looks like you got the page title instead of the URL... see if you can track down the original URL and update.